Gemfile.lock

```diff
@@ -106,13 +105,14 @@ GEM
     zlib (2.1.1)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
I just realized we had Gemfile.lock into version control when we shouldn't - please rebase from master, I just removed that.